releases/mainnet/1.2.9/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.2.9</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.2.9</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.9/contracts.json">Contract addresses</a></li>
+				<li><a href="">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.2.9/abi/index.html">ABIs</a></li>
+				<li>Links to GitHub repositories</li>
+					<ul>
+						DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.9" target="_blank">changelog</a>)</li>
```

Comment:
Release is not there yet. We need to either comment out this part or wait for DSS deploy script release to be published before merging.

Reply:
In general we are publishing this before the `dss-deploy-scripts` is done but in this specific case I don't think we will have a `1.2.9` in that repo as there won't be any changes. For now I'm keeping this commented.